utils/build-presets.ini

```diff
@@ -1102,6 +1108,12 @@ relocate-xdg-cache-home-under-build-subdir
 # to avoid build failures
 extra-cmake-options=
   -DBUILTINS_CMAKE_ARGS:STRING="-DCAN_TARGET_i386:BOOL=OFF"
+# This is to prevent failures in linking
+# tsan_interceptors_dispatch as part of libclang_rt.tsan
+# We are now inheriting the -Wl,-z,defs linker flags
+# from HandleLLVMOptions.cmake, so the compiler now complains
+# for BlocksRuntime symbols that are undefined
+  -DCOMPILER_RT_INTERCEPT_LIBDISPATCH:BOOL=OFF
```

Comment:
This should not be turned off, otherwise we will break TSan. @edymtt 

Can we fix the undefined references by linking the blocks runtime `-fblocks`?

The CMake code here is trying to do this:
https://github.com/llvm/llvm-project/blob/f418f88824905c372cb9252b3f675012a3fc799d/compiler-rt/CMakeLists.txt#L172

Reply:
I can try explore that -- to the best of my understanding that is not present in CI nodes, and we are building that ourselves when we are building Swift, but likely I'm missing something